### PR TITLE
MSTest Playwright Legacy sample

### PR DESCRIPTION
**Pull request by Baaryan**

### CODE REVIEW

File: .github/workflows/sanity_workflow.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            repo: context.repo.repo,
+            name: process.env.job_name,
+            head_sha: process.env.commit_sha,
+            status: 'completed',
+            conclusion: process.env.conclusion
+            }).catch((err) => ({status: err.status, response: err.response}));
+            console.log(`The status-check response : ${result.status} Response : ${JSON.stringify(result.response)}`)
+            if (result.status !== 201) {
+              console.log('Failed to create check run')
+            }
```
Comment (kamal-kaur04) [inline .github/workflows/sanity_workflow.yml]: new line

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+*.binlog
+
+# NVidia Nsight GPU debugger configuration file
+*.nvuser
+
+# MFractors (Xamarin productivity tool) working folder
+.mfractor/
+
+# Local History for Visual Studio
+.localhistory/
```
Comment (kamal-kaur04) [inline .gitignore]: new line

File: MSTest-Playwright-BrowserStack/parallel.conf.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    },
+    "playwright-webkit": {
+      "browser": "playwright-webkit"
+    },
+    "playwright-firefox": {
+      "os": "OS X",
+      "browser": "playwright-firefox"
+    }
+  }
+}
```
Comment (kamal-kaur04) [inline MSTest-Playwright-BrowserStack/parallel.conf.json]: new line

File: MSTest-Playwright-BrowserStack/single.conf.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    "name": "BStack Playwright single test",
+    "browserstack.debug": true
+  },
+
+  "environments": {
+    "chrome": {
+      "browser": "chrome"
+    }
+  }
+}
```
Comment (kamal-kaur04) [inline MSTest-Playwright-BrowserStack/single.conf.json]: new line

**Review by kamal-kaur04 — CHANGES_REQUESTED** [review]
please add source capability in this PR also

**Comment by Baaryan** [comment]
Done

File: MSTest-Playwright-BrowserStack/single.conf.json [hunk 5]
```diff
@@ -5,6 +5,7 @@
   "capabilities": {
     "build": "browserstack-build-1",
     "name": "BStack Playwright single test",
+    "source": "mstest-playwright:sample-main:v1.0",
```
Comment (kamal-kaur04) [inline MSTest-Playwright-BrowserStack/single.conf.json]: browserstack.source

**Review by kamal-kaur04 — CHANGES_REQUESTED** [review]

File: .github/workflows/sanity_workflow.yml [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
```
Comment (kamal-kaur04) [inline .github/workflows/sanity_workflow.yml]: new line

**Review by kamal-kaur04 — CHANGES_REQUESTED** [review]

File: .github/workflows/sanity_workflow.yml (hunk 6 above)
Comment (Baaryan) [reply .github/workflows/sanity_workflow.yml]: Should I add another new line at the end or inside the if condition?

Comment (Baaryan) [reply .github/workflows/sanity_workflow.yml]: Done

**Review by RutvikChandla — APPROVED** [review]

**Pull request merged**